recipes/pygdp/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "2.0.1" %}
+
+package:
+    name: pygdp
+    version: {{ version }}
+
+source:
+    fn: v{{ version }}.tar.gz
+    url: https://github.com/USGS-CIDA/pyGDP/archive/v{{ version }}.tar.gz
+    sha256: fb64a5d665a5218b2bc105beb768ece199d21166c99a445a0792c372ba2e99a3
+
+build:
+    number: 0
+    skip: True  # [py3k]
+    script: python setup.py install --single-version-externally-managed  --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - owslib
+    run:
+        - python
+        - owslib
+        - python-dateutil
+        - lxml
+test:
+    imports:
+        - pyGDP
+
+about:
+    home: https://github.com/USGS-CIDA/pyGDP
+    license: Public Domain
+    summary: Python interface to facilitate working with USGS-CIDA Geo Data Portal web services
+
+extra:
+    recipe-maintainers:
+        - ocefpaf
+        - ayan-usgs
```

Comment:
:+1: 

PS: If you want to be a maintainer of any other recipe in [conda-forge](https://conda-forge.github.io/feedstocks.html) just send a PR with your GitHub handle in the `extra/recipe-maintainers` field and you'll be automatically added.